apps/docs-snippets2/src/encoding/working-with-bytes.ts

```diff
@@ -0,0 +1,154 @@
+// #region full
+import { randomBytes } from 'crypto';
+import {
+  ArrayCoder,
+  B256Coder,
+  B512Coder,
+  BigNumberCoder,
+  BooleanCoder,
+  EnumCoder,
+  NumberCoder,
+  RawSliceCoder,
+  StdStringCoder,
+  StringCoder,
+  StructCoder,
+  TupleCoder,
+  VecCoder,
+  hexlify,
+} from 'fuels';
+
+// #region working-with-bytes-1
+const u8Coder = new NumberCoder('u8');
+const encodedU8 = u8Coder.encode(255);
+console.assert(
+  encodedU8.toString() === new Uint8Array([255]).toString(),
+  'Encoded U8 should be equal to 255'
+);
+
+const u16Coder = new NumberCoder('u16');
+const encodedU16 = u16Coder.encode(255);
+console.assert(
+  encodedU16.toString() === new Uint8Array([0, 255]).toString(),
+  'Encoded U16 should be equal to 255'
+);
+
+const u32Coder = new NumberCoder('u32');
+const encodedU32 = u32Coder.encode(255);
+
+console.assert(
+  encodedU32.toString() === new Uint8Array([0, 0, 0, 255]).toString(),
+  'Encoded U32 should be equal to 255'
+);
+
+const u64Coder = new BigNumberCoder('u64');
+const encodedU64 = u64Coder.encode(255);
+console.assert(
+  encodedU64.toString() === new Uint8Array([0, 0, 0, 0, 0, 0, 0, 255]).toString(),
+  'Encoded U64 should be equal to 255'
+);
+
+const u256Coder = new BigNumberCoder('u256');
+const encodedU256 = u256Coder.encode(255);
+console.assert(
+  encodedU256.toString() ===
+    new Uint8Array([
+      0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0,
+      255,
+    ]).toString(),
```

Comment:
@maschad What is the motivation behind using `console.asset` here?

This approach has increased the verbosity of some code snippets.

Before:
<img width="697" alt="Screenshot 2024-10-02 at 10 05 57" src="https://github.com/user-attachments/assets/31907cfb-b890-48e0-ad85-fb94201f9248">

After:
<img width="692" alt="Screenshot 2024-10-02 at 10 06 31" src="https://github.com/user-attachments/assets/d12a6c93-9cd3-4459-9d6d-5fb3ecef7f77">

Reply:
In fact, we should not use `console.assert` or `expect` ([here](https://github.com/FuelLabs/fuels-ts/pull/3233/files#diff-2ade8bacc0cbacf42b1a485e15be9fd666c0f0ec0c3f57ce64dbc2026547604eR57-R58)) in the snippets anymore.

Only `console.log` and those will be converted to test assertions.

Let's do another quick sync over this? @maschad @Torres-ssf

Reply:
@arboleya Sure!

While working on [FuelLabs/fuels-ts#3233](https://github.com/FuelLabs/fuels-ts/pull/3233), I aimed to keep the code snippets as they are. However, I'm happy to remove the test `expects` if that’s the preferred direction.

Reply:
We also had this issue to remove assertions from the snippets:
 - https://github.com/FuelLabs/fuels-ts/issues/1138

I [closed it](https://github.com/FuelLabs/fuels-ts/issues/1138#issuecomment-2373700450), but the rationale remains for the new snippets.